packages/client/components/ActivityLibrary/ActivityLibraryCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+interface Props {
+  name: string
+  type: string
+  teamName: string
+}
+
+const ActivityLibraryCard = (props: Props) => {
```

Comment:
I don't foresee us using this full component in the `/meetings` dashboard, since it will eventually have onClick handlers, etc. for entering the activity details view. 

We can refactor the relevant parts of this into its own component once we add the UI portion of this to the dashboard.

Reply:
will allow myself to nitpick a little

1. Parts of this component should be reusable as they're exactly the same (ie image part with this rounded thingy in the corners)
<img width="355" alt="Screenshot 2023-03-09 at 07 23 11" src="https://user-images.githubusercontent.com/1017620/223938012-a32e9afd-d043-45c1-8d88-fca6fee56448.png">

2. imo activity details should be a route like `/activity-library/:activity-id` which preferably makes this component a link (this also will be good for accessibility and keyboard navigation reasons), see the video below

https://user-images.githubusercontent.com/1017620/223938475-b73a31c0-7f62-47b2-ba18-0221a742ac96.mp4

if it's a link we can easily use tab + enter to navigate that view, which is a win.

it's not really related to this pr though 🙃